e2e/host/txtdirect.config

```diff
@@ -1,9 +1,9 @@
 :80 {
-    txtdirect {
-        enable host path
-        resolver 172.20.10.1:53
-        logfile stdout
+    route {
+        txtdirect {
+            enable host path
+            resolver 172.20.10.1:53
+            logfile stdout
+        }
```

Comment:
It's because of the directives order in v2: https://caddyserver.com/docs/caddyfile/directives#directive-order
Options are:
[`order`](https://caddyserver.com/docs/caddyfile/options) option in global settings
[`route`](https://caddyserver.com/docs/caddyfile/directives/route) directive

Reply:
Ok so we are breaking compatibility so good time to revamp more stuff as we are doing already.